README.md

```diff
@@ -20,34 +20,35 @@ Based on our best practices for deployment and configuration, [Percona Operator
 * Customize MySQL configuration
 * Manage system user passwords
 
+You interact with Percona Operator mostly via the command line tool. If you feel more comfortable with operating the Operator and database clusters via the web interface, there is [Percona Everest](https://docs.percona.com/everest/index.html) - an open-source web-based database provisioning tool available for you. It automates day-to-day database management operations for you, reducing the overall administrative overhead. [Get started with Percona Everest](https://docs.percona.com/everest/quickstart-guide/quick-install.html).
+
 ## Status
 
 **This project is in the tech preview state right now. Don't use it on production.**
 
 As of today, we recommend using [Percona Operator for MySQL based on Percona XtraDB Cluster](https://docs.percona.com/percona-operator-for-mysql/pxc/index.html), which is production-ready and contains everything you need to quickly and consistently deploy and scale MySQL clusters in a Kubernetes-based environment, on-premises or in the cloud.
 
-## Roadmap
-
-We have an experimental public roadmap which can be found [here](https://github.com/percona/roadmap/projects/1). Please feel free to contribute and propose new features by following the roadmap [guidelines](https://github.com/percona/roadmap).
-
 ## Architecture
 
 Percona Operators are based on the [Operator SDK](https://github.com/operator-framework/operator-sdk) and leverage Kubernetes primitives to follow best CNCF practices.
 
-## Installation
+## Documentation
+
+To learn more about the Operator, check the [Percona Operator for MySQL documentation](https://docs.percona.com/percona-operator-for-mysql/ps/index.html).
 
-It usually takes two steps to deploy Percona Server for MySQL on Kubernetes:
+# Quickstart installation
 
-* Deploy the operator from `deploy/bundle.yaml`
```

Comment:
Commands could be added here, such as:

`kubectl apply -f deploy/bundle.yaml`

I think it would be better to provide the command for them to execute

Reply:
Updated